src/clandinin_lab_to_nwb/brezovec/brezovec_convert_session.py

```diff
@@ -23,29 +22,16 @@ def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str,
     source_data = dict()
     conversion_options = dict()
 
-    # Add Recording
-    source_data.update(dict(Recording=dict()))
-    conversion_options.update(dict(Recording=dict()))
-
-    # Add LFP
-    source_data.update(dict(LFP=dict()))
-    conversion_options.update(dict(LFP=dict()))
-
-    # Add Sorting
-    source_data.update(dict(Sorting=dict()))
-    conversion_options.update(dict(Sorting=dict()))
-
-    # Add Behavior
-    source_data.update(dict(Behavior=dict()))
-    conversion_options.update(dict(Behavior=dict()))
+    # Add Fictrac
+    file_path = data_dir_path / "fictrac" / "fictrac-20200228_161226.dat"
+    source_data.update(dict(FicTrac=dict(file_path=str(file_path))))
+    conversion_options.update(dict(FicTrac=dict()))
```

Comment:
Do you need to add an empty dict here?

Reply:
No, inherited from the cookiecutter. Removing.